app/policies/user_context.rb

```diff
@@ -1,9 +1,14 @@
 class UserContext
   attr_reader :user, :params, :controller_name
+  attr_reader :catalog_access
 
   def initialize(user, params, controller_name)
     @user = user
     @params = params
     @controller_name = controller_name
   end
+
+  def catalog_access
+    @catalog_access ||= Insights::API::Common::RBAC::Access.new('catalog,approval')
```

Comment:
@eclarizio The parameter should be an empty string "" for now I am waiting for the RHCLOUD team to fix it so we can pass in a comma delimited list of apps to fetch the permissions from.